.circleci/config.yml

```diff
@@ -108,6 +108,13 @@ commands:
       - *install-assets
       - *save-assets
 
+  install-puppeteer:
+    steps:
+      - run:
+          name: Install Puppeteer with Chromium
+          command: |
+            yarn add puppeteer@19.2.0
```

Comment:
do we need to lock it to 19.2.0. [example from civil apply](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/62f68bd05d19d90472885c6e63621cc50b18b246/.circleci/config.yml#L196)

Reply:
Have removed